Sources/BrowserServicesKit/Common/Extensions/NSViewControllerExtension.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  NSViewControllerExtension.swift
+//
+//  Copyright © 2021 DuckDuckGo. All rights reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+#if !os(iOS)
+import Cocoa
+import Foundation
+
+extension NSViewController {
```

Comment:
Is any of this extension used within BSK? I'm wary about having such platform specific code here, and I don't see this being used so I think this file can be binned. 🙂

Reply:
Confirmed binable thanks :)